dev/.rat-excludes

```diff
@@ -15,3 +15,5 @@ RELEASE
 build
 .*gradle
 .*json
+.helmignore
+.*yml
```

Comment:
What do you want to exclude here? This PR adds `.yaml` file instead of `yml`, doesn't it?

In addition, AFAIK, you added the license header already. Does this mean the added header (of this file) is incompatible with Apache RAT check?